src/app/pages/partners/results/results.tsx

```diff
@@ -0,0 +1,300 @@
+import React from 'react';
```

Comment:
Again a side-by-side with results.js could be helpful.

Reply:
It was `git mv`d in the first commit, but the number of changes made git treat it as delete and create.